packages/plugin-infra/src/site-adaptor/context.ts

```diff
@@ -4,16 +4,20 @@
 import type { Subscription } from 'use-subscription'
 import type { IdentityResolved } from '../types.js'
 import { __setUIContext__, type __UIContext__ } from '../dom/context.js'
+import type { NextIDPlatform, PersonaIdentifier } from '@masknet/shared-base'
 
 export * from '../dom/context.js'
 export interface __SiteAdaptorContext__ extends __UIContext__ {
     lastRecognizedProfile: Subscription<IdentityResolved | undefined>
     currentVisitingProfile: Subscription<IdentityResolved | undefined>
+    currentNextIDPlatform: NextIDPlatform | undefined
+    currentPersonaIdentifier: Subscription<PersonaIdentifier | undefined>
 }
 export let lastRecognizedProfile: __SiteAdaptorContext__['lastRecognizedProfile']
 export let currentVisitingProfile: __SiteAdaptorContext__['currentVisitingProfile']
+export let currentNextIDPlatform: NextIDPlatform | undefined
+export let currentPersonaIdentifier: __SiteAdaptorContext__['currentPersonaIdentifier']
```

Comment:
@Jack-Works 
`currentPersonaIdentifier` seems never gets initialized.

<img width="1061" alt="image" src="https://github.com/DimensionDev/Maskbook/assets/1141198/82bf8fe5-635c-43fa-8976-581d4f424c42">


It will fail this

<img width="976" alt="image" src="https://github.com/DimensionDev/Maskbook/assets/1141198/85cbc76c-95e5-4fd5-86b6-49412676b613">

Reply:
ack, let me fix this

Reply:
please try this: #10817